Source/WebCore/platform/mediastream/mac/AVVideoCaptureSource.h

```diff
@@ -158,6 +158,10 @@ class AVVideoCaptureSource : public RealtimeVideoCaptureSource, private Orientat
     Timer m_verifyCapturingTimer;
     uint64_t m_framesCount { 0 };
     uint64_t m_lastFramesCount { 0 };
+
+#if PLATFORM(IOS) || PLATFORM(VISION)
+    std::optional<int> m_presentingApplicationPID;
```

Comment:
This seems like the wrong place to store this value. WE should be storing the presenting PID in the GPU process, on a per GPUToWebContentConnection basis.

Reply:
Done